src/renderer/pages/JobList.tsx

```
@@ -66,15 +73,72 @@ export default function JobList() {
+    switch (filter) {
+      case 'remote':
+        // Matches: "100% remote", "remote", "vollständig remote", "full remote"
+        return normalized.includes('100%') ||
+               normalized === 'remote' ||
+               normalized.includes('vollständig') ||
+               normalized.includes('full remote');
+      case 'hybrid':
+        // Matches: "hybrid", or percentage values that are not 100% or 0%
+        if (normalized.includes('hybrid')) return true;
+        // Check for partial remote percentages (e.g., "50% remote", "80%")
+        const percentMatch = normalized.match(/(\d+)%/);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)